global_frame: robot_0/odom
robot_base_frame: robot_0/base_link
update_frequency: 2.5
publish_frequency: 2.5
map_topic: map
subscribe_to_updates: true
always_send_full_costmap: true
#set if you want the voxel map published
publish_voxel_map: true

#set to true if you want to initialize the costmap from a static map
static_map: false

#begin - COMMENT these lines if you set static_map to true
rolling_window: true
width: 3
height: 3
origin_x: -1.5
origin_y: -1.5

resolution: 0.5
#end - COMMENT these lines if you set static_map to true

map_type: costmap
track_unknown_space: false

transform_tolerance: 0.25
obstacle_range: 6.0
min_obstacle_height: 0.0
max_obstacle_height: 2.0
raytrace_range: 8.5

robot_radius: 0.185
inflation_radius: 1.0
cost_scaling_factor: 2.0
lethal_cost_threshold: 100
observation_sources: scan
scan: {data_type: LaserScan, topic: base_scan, expected_update_rate: 0.4, marking: true, clearing: true}
